### PR TITLE
Analyze spec

### PR DESCRIPTION
**Pull request by marccampbell**

### CODE REVIEW

File: cmd/analyze/cli/run.go [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+		} else if analyzeResult.IsFail {
+			fmt.Printf("Fail: %s\n %s\n", analyzeResult.Title, analyzeResult.Message)
+		}
+	}
+
+	return nil
+}
+
+func isURL(str string) bool {
+	_, err := url.ParseRequestURI(str)
```
Comment (divolgin) [inline cmd/analyze/cli/run.go]: pretty sure this parses absolute file paths

Comment (divolgin) [reply cmd/analyze/cli/run.go]: you should just check if file exists

Comment (marccampbell) [reply cmd/analyze/cli/run.go]: this is elsewhere in the code...  seems like a bigger issue?

**Review by divolgin — APPROVED** [review]

File: cmd/analyze/cli/run.go (hunk 1 above)
Comment (divolgin) [reply cmd/analyze/cli/run.go]: yeah.... I see it

**Pull request merged**